using-timescaledb/data-retention.md

```diff
@@ -20,6 +26,68 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the hypertable as well as the continuous aggregate. Given a hypertable
+`conditions` and a continuous aggregate `conditions_summary` over the
+hypertable:
+
+```sql
+CREATE VIEW conditions_summary WITH (timescaledb.continuous) AS
+SELECT location,
+       time_bucket('1d', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY location, time_bucket('1d', "time")
+```
+
+Data retention on a continous aggregate will normally not propagate to
```

Comment:
This doesn't continue the sentence started before the view example as one would expect.